testing/agent_test.py

```diff
@@ -66,316 +67,34 @@ def test_fixed_plan_agents(self):
         self.assertEqual(time_taken, 10)
         self.assertEqual(env.mdp.get_standard_start_state().player_positions, end_state.player_positions)
 
-    def test_two_coupled_agents(self):
-        a0 = CoupledPlanningAgent(self.mlp_large)
-        a1 = CoupledPlanningAgent(self.mlp_large)
-        agent_pair = AgentPair(a0, a1)
-        start_state = OvercookedState(
-            [P((2, 2), n),
-             P((2, 1), n)],
-            {}, all_orders=[])
-        env = OvercookedEnv.from_mdp(large_mdp, start_state_fn=lambda: start_state)
-        trajectory, _, reward, _ = env.run_agents(agent_pair, include_final_state=True, display=DISPLAY)
-        print(reward)
-        self.assertEqual(reward, 20)
+    # Deprecated. Due to CoupledPlanningAgent
+    # def test_two_coupled_agents(self):
+    #     a0 = CoupledPlanningAgent(self.mlp_large)
+    #     a1 = CoupledPlanningAgent(self.mlp_large)
+    #     agent_pair = AgentPair(a0, a1)
+    #     start_state = OvercookedState(
+    #         [P((2, 2), n),
+    #          P((2, 1), n)],
+    #         {})
+    #     env = OvercookedEnv.from_mdp(large_mdp, start_state_fn=lambda: start_state)
+    #     trajectory, time_taken, _, _ = env.run_agents(agent_pair, include_final_state=True, display=DISPLAY)
+    #     end_state = trajectory[-1][0]
+
```

Comment:
We can probably remove deprecated tests too. My main philosophy for keeping commented out deprecated stuff is to indicate that "at one point, this functionality existed. If you want to actually resume the functionality, you should probably go back in the commit history and find everything related to this feature, copy it back into the code, and update it". That's why i usually recommend deleting minor things in the middle of other code / testing, but tend to try to keep commented out classes or class methods.

Reply:
Sounds good